### ISSUE TITLE
Could you share the full set of best hyperparameters somewhere?

### ISSUE THREAD

**Issue by hepengfe** [facebookresearch/CMR#7]
For example, what's the best `l2d` for all experiments? There could be other missing hyper-parameters and missing experiments, and could you help check them and provide the configurations as well?

**Comment by hepengfe** [comment]
Also, what's the metric to select the best hyper-parameter?
